routes/admin-login.js

```diff
@@ -1,8 +1,54 @@
 var express = require('express');
 var router = express.Router();
+var firebase = require('firebase');
+var config = require('./config.js');
+
+if(!firebase.apps.length)
+	firebase.initializeApp(config.config);
 
 router.get('/', function(req, res, next) {
 	res.render('admin-login');
 });
 
+router.post('/', signIn, function(req, res) {
+	//checks if user is signed in
+	if(firebase.auth().currentUser)
```

Comment:
to me, `currentUser` implies this auth service can only handle one user at a time, and that along with the client side code in the Firebase docs makes me think Firebase Auth is intended for clients